src/v3/.eslintrc.js

```diff
@@ -307,7 +313,18 @@ module.exports = {
             toBeFalsy: 'Avoid `toBeFalsy`',
           },
         ],
-
+        'testing-library/no-wait-for-multiple-assertions': 'warn',
+        'testing-library/no-node-access': 'warn',
+        'testing-library/no-container': 'warn',
+        'testing-library/prefer-screen-queries': 'off',
+        'testing-library/render-result-naming-convention': 'off',
+        'testing-library/prefer-explicit-assert': [
+          'warn',
+          {
+            assertion: 'toBeInTheDocument',
+            includeFindQueries: true,
+          },
+        ],
```

Comment:
I don't know how I ended up here, got carried away in a yak shave. Result is I refactored and updated some eslint configs. I can back these out if we think it's too much or is distracting.